app/CMakeLists.txt

```diff
@@ -30,9 +30,9 @@ if (CCF_UNSAFE)
     message(FATAL_ERROR "The unsafe CCF variant is only available on SGX builds.")
   endif()
 
-  find_package(ccf_sgx_unsafe 4.0.17 REQUIRED)
+  find_package(ccf_sgx_unsafe 5.0.0 REQUIRED)
```

Comment:
IIUC, unsafe SGX builds are no longer supported in CCF 5. Do we need this?

Reply:
All the unsafe stuff can be dropped.

Reply:
Should I just remove this line?

Reply:
Done